### DISCUSSION TITLE
Does Eclipse Milo Client SDK Support SCAN to Collect Data

### DISCUSSION THREAD

**Discussion by jaikishansharma** [eclipse-milo/milo#1018] [General]
Similar to subscription do we have support for SCAN in the milo SDK client to collect the data? For SCAN, the timer will be at the client-side; for UA-Server, it is like a demand read. Please share more information about classes get used for SCAN.

**Comment by kevinherron** [comment]
SCAN isn’t an actual OPC UA concept, but it sounds like you’re talking about issuing a periodic call to the Read service.

You can do this, of course, but you’ll have to set up the timer yourself.

**Reply by jaikishansharma** [reply]
Thanks, Kevin for your clarification. I think we have to implement the scheduler to achieve this and utilized the Read service for the same to support SCAN on top of Milo SDK.
